scripts/start.sh

```diff
@@ -64,23 +64,24 @@ fi
 # Argument passed to dev_appserver.py to indicate whether or not to
 # clear the datastore.
 CLEAR_DATASTORE_ARG="--clear_datastore=true"
-# Argument passed to gulpfile.js to help build with minification.
-MINIFICATION=false
+
+# Argument passed to feconf.py to help choose production templates folder.
+FORCE_PROD_MODE=False
 for arg in "$@"; do
   if [ "$arg" == "--save_datastore" ]; then
     CLEAR_DATASTORE_ARG=""
   fi
   # Used to emulate running Oppia in a production environment.
   if [ "$arg" == "--prod_env" ]; then
-    MINIFICATION=true
+    FORCE_PROD_MODE=True
```

Comment:
Ditto

Reply:
@seanlip FORCE_PROD_MODE is a string in sh script which contains the python boolean value so  i guess it should be python way. however there was a type in line 116. 

Pushing the changes, actually 'run_e2e_tests.sh --suite=misc --prod_env' passes on my machine but not onto travis.